Sources/FuzzilliCli/main.swift

```diff
@@ -62,7 +62,6 @@ Options:
     --noAbstractInterpretation  : Disable abstract interpretation of FuzzIL programs during fuzzing. See
                                   Configuration.swift for more details.
     --collectRuntimeTypes       : Collect runtime types of variables in program when interesting one found.
```

Comment:
Unrelated to this PR, but can you make this something like "Collect runtime type information for programs that are added to the corpus"? I think that's a bit more clear